src/postgres/connectToPostgresClient.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Client } from 'pg';
+
+export async function connectToPostgresClient(clientConfig): Promise<Client> {
+
+    const client = new Client(clientConfig);
+    try {
+        return client;
+    } catch (err) {
+        const error = <{ message?: string, name?: string }>err;
```

Comment:
I'm guessing you copied this function from somewhere else (mongo?) but it's actually not doing anything for Postgres. You're catching the error and immediately throwing it. You can remove the try/catch block and since you're only left with one line of code I would just get rid of `connectToPostgresClient` entirely